secator/hooks/api.py

```
@@ -0,0 +1,185 @@
+		response.raise_for_status()
+		result = response.json()
+		debug(f'API response: {result}', sub='hooks.api', verbose=True)
+		return result
+	except requests.exceptions.RequestException as e:
+		logger.error(f"API request failed: {e}")
+		debug(f'API error: {e}', sub='hooks.api')
+		return None
+
+
+def update_runner(self):
+	"""Update runner state via API."""
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)